components/06-examples/machtigen.hbs

```diff
@@ -0,0 +1,78 @@
+{{> '@header' }}
+<main>
+  <div>
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <div class="row">
+          <div class="col-md-12">
+            <h1>Inloggen</h1>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-md-12">
+            <ul class="nav nav-tabs">
+              <li class="active">
+                <a href="machtigen.html">Voor mezelf</a>
+              </li>
+              <li >
+                <a href="machtigen1.html">Voor een ander</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div class="row dso-rich-content">
```

Comment:
`.dso-rich-content` is een eigen component en mag niet op `.row` worden geplaatst. Waarom wil je dit?